src/brains2/CMakeLists.txt

```diff
@@ -31,6 +32,9 @@ ament_auto_find_build_dependencies()
 # find other libraries: eigen3, nlohmann json
 find_package(Eigen3 REQUIRED)
 find_package(nlohmann_json REQUIRED)
+find_package(yaml-cpp REQUIRED)
+find_package(acados REQUIRED PATHS ${CMAKE_INSTALL_PREFIX}/../acados/cmake)
+include_directories(install/acados/include)  # hacky way to also find acados_c headers
```

Comment:
🫨

Reply:
This may actually not be needed anymore (now that the `find_package(acados)` actually works)